Tests/Utilities/Mocks/Provider/MockDeeplinkAbilityProvider.swift

```diff
@@ -0,0 +1,10 @@
+@testable import PrimerSDK
+import Foundation
+
+struct MockDeeplinkAbilityProvider: DeeplinkAbilityProviding {
+    var isDeeplinkAvailable: Bool = true
```

Comment:
suggestion: redundant bool here
```suggestion
    var isDeeplinkAvailable = true
```

Reply:
we can use implicit return here